app.js

```diff
@@ -17,6 +19,9 @@ app.use((err, req, res, next) => {
         if (err.code === '22P02'){
             res.status(400).send('Invalid Input!')
         }
+        if (err.code === 500){
```

Comment:
The 500 error is a bit of a special one.
We don't want it to be sent out when there is a specific error code, usually we want it to be sent out when we have handled all other errors and there is still an error happening! 
So it should just be the default for this app.use in case no other conditional statement is true.